### ISSUE TITLE
IE Alt K opens Feedback Hub

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#346]
**Test device**
Dell
**Operating System**
Win 0
**Browser**
IE
**Problem description**
For https://github.com/phetsims/QA/issues/503. IE is being dropped, but I'm still doing a basic check to see if everything opens and works, which is how I caught this. May need to be discussed.
In IE, when using keyboard nav, the Alt + K command opens up the feedback hub window, rather than pause the sim. Alt + L works fine. This may not be worth fixing, as IE is not going to be supported in future, but it may be worth detecting IE and changing the keyboard nav dialog.

**Visuals**
![IEFeedbackhub](https://user-images.githubusercontent.com/41024075/82686936-daf4a800-9c13-11ea-8b2c-e350f08a36c8.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
  
Name: ‪Molecules and Light‬

URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.50/phet/molecules-and-light_all_phet.html?zoom=true

Version: 1.5.0-dev.50 2020-05-12 15:11:56 UTC

Features missing: touch, fullscreen

Flags: pointerEnabled, msPointerEnabled, pixelRatioScaling

User Agent: Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; Touch; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Tablet PC 2.0; Zoom 3.6.0; rv:11.0) like Gecko

Language: en-US

Window: 1536x750

Pixel Ratio: 2.5/1

WebGL: WebGL 0.94

GLSL: WebGL GLSL ES 0.94

Vendor: Microsoft (Internet Explorer)

Vertex: attribs: 16 varying: 15 uniform: 512

Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)

Max viewport: 16384x16384

OES_texture_float: true

Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe  - Ill see if this can be prevented, in case we want to do that.  Is this specific to IE11 or does it also happen in Edge?

**Comment by KatieWoe** [comment]
It works fine on chromium Edge. Don't have old version of Edge to see if it does there.

**Comment by jessegreenberg** [comment]
OK no worries, thats good enough and indicates that this isn't a microsoft reserved key command still. Given low use for IE11 and the fact that the world is phasing it out, I think it would be acceptable to try and prevent IE11 from opening the feedback hub if we can.

**Comment by jessegreenberg** [comment]
OK, I tried to prevent this from opening and could not with event.preventDefault(). This makes sense, it doesn't seem like anything a web developer should be able to control. Would be good to discuss with @terracoda @Matthew-Moore240 and maybe others to see if this should prevent use of Alt + K.

**Comment by terracoda** [comment]
I feel we can leave as is. The is an added feature and does not prevent use of the Pause/Play button. Alt + K is particularly useful for non-visual access and non-visual users will likely not be using IE11.

**Comment by terracoda** [comment]
@jessegreenberg will add this to a11y bugs list.

**Comment by jessegreenberg** [comment]
Added to the list, closing.
